cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/rolling/RackRollingTest.java

```diff
@@ -1021,11 +1021,15 @@ public void shouldRestartCombinedNodesInExpectedOrderAndBatched() throws Executi
 
         // The expected order to restart is batches of nodes that do not have partitions in common
         // starting with the largest batches and then the broker that is the active controller
```

Comment:
can we update the comment since we are not doing batched restarts for combined nodes?